CHANGELOG.md

```diff
@@ -1,5 +1,34 @@
 # Change Log
 
+## [v1.1.7](https://github.com/ably/ably-ruby/tree/v1.1.7)
+
+[Full Changelog](https://github.com/ably/ably-ruby/compare/v1.1.6...HEAD)
+
+**Implemented enhancements:**
+
+- Implement RSC7d \(Ably-Agent header\) [\#230](https://github.com/ably/ably-ruby/issues/230)
+- Support Ruby 3.x [\#220](https://github.com/ably/ably-ruby/issues/220)
+
+**Closed issues:**
+
+- Make CI Green [\#254](https://github.com/ably/ably-ruby/issues/254)
```

Comment:
I would remove entries like this from issues and pull requests lists, which don't present any immediate customer-facing value. There is always the full changelog link if people are interested in every issue and pull request.

Reply:
Agree with @QuintinWillison

Reply:
done - d65eb00